### ISSUE TITLE
Loss Function Calculation

### ISSUE THREAD

**Issue by ZhuHouYi** [vandit15/Self-Supervised-Gans-Pytorch#2]
Hello, I have a question. I found in your code that you use WGAN loss function, which is different from the loss function used in the original paper, and you use `torch.sum()` operation instead of `torch.mean()` operation to calculate the loss for a batch of samples, may I ask why you do so

**Comment by ZhuHouYi** [comment]
Because when I ran the code, I found that `torch.sum()` would calculate a particularly large loss, the value of the penalty term and rotation loss would be insignificant relative to that loss

**Comment by bizarrelovetriangle** [comment]
> Because when I ran the code, I found that `torch.sum()` would calculate a particularly large loss, the value of the penalty term and rotation loss would be insignificant relative to that loss

Did you get any good result from this implementation?
I'm thinking about implementing SSGAN or SAGAN

P.S. thank you for pointing out this is a WGAN loss.
